### ISSUE TITLE
wrong CTF files names with write_raw_bids

### ISSUE THREAD

**Issue by romquentin** [mne-tools/mne-bids#161]
Hi, 
`write_raw_bids` does not copy the new BIDS name of the CTF folder (i.e. sub-01_ses-01_task-01.ds) for each of the files inside the folder (like .res4 or .meg4)
However, when reading raw CTF file with `mne.read_raw_ctf`, the name of the .ds folder need to be the same that the files that are in this .ds folder.
Is it possible to make `write_raw_bids` not only copy the files in the .ds folder but also change the name of these files to reflect the new name of the .ds folder?
I can try to make a PR.
Thank you, 
Romain

**Comment by jasmainak** [comment]
@romquentin yes I found this issue too in [my PR](https://github.com/mne-tools/mne-bids/pull/135/files#r243751864). Help in fixing this would be more than welcome!
